### PR TITLE
edgecontroller: improve test coverage for edgecontroller/manager/location.go

### LINKED ISSUE #2239: cloud/pkg/edgecontroller/manager: improve the test coverage
Add more tests to import test coverage. Details refer to #2110 .

### PR DESCRIPTION
**Pull request by gccio**
**What type of PR is this?**

**What this PR does / why we need it**:
imporove coverage of edgecontroller/manager/location.go

**Which issue(s) this PR fixes**:

Fixes #2239 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloud/pkg/edgecontroller/manager/location_test.go [hunk 1]
```diff
@@ -206,3 +206,789 @@ func TestDeleteSecret(t *testing.T) {
...
+			exist:    false,
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			if got, exist := test.lc.GetNodeStatus(test.nodeName); !reflect.DeepEqual(got, test.want) || !reflect.DeepEqual(exist, test.exist) {
+				t.Errorf(
+					"Manager.TestGetNodeStatus() case failed: gotStatus = %v,gotExist = %v, wantStatus = %v.  wantExist: %v",
+					got, exist, test.want, test.exist,
```
Comment (fisherxu) [inline cloud/pkg/edgecontroller/manager/location_test.go:288]: Delete the extra line breaks here :)

**Review by fisherxu — COMMENTED** [review]
Thanks for the PR

File: cloud/pkg/edgecontroller/manager/location_test.go (hunk 1 above)
Comment (gccio) [reply cloud/pkg/edgecontroller/manager/location_test.go]: deleted

**Pull request merged**